app/src/main/res/layout/fragment_character.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ProgressBar
```

Comment:
you don't need the visibility=gone?

Reply:
No, because when i call the fragment dialog always show the progress bar then yes i call a method how set the visibility in gone. 
That is what i think